docs/workgroup/workgroup_charter.rst

```diff
@@ -107,27 +107,27 @@ tool.
 Budget
 ------
 
-The working group will not require a dedicated budget in its initial phase.
-If needed, the working group may ask the board of directors for any funds needed
+The Editorial Board will not require a dedicated budget in its initial phase.
+If needed, the Editorial Board may ask the PSF board of directors for any funds needed
 (for meetings, writers, translators, software development, etc.).
 
 Participants
 ------------
-The Python Steering Council are permanent members of this working group.
+The Python Steering Council are permanent members of the Editorial Board.
 The initial workgroup members include:
 
 - Carol Willing
 - Ned Batchelder
 - Mariatta Wijaya
 - Julien Palard (Lead, Infrastructure/Localization)
 
-The workgroup may expand to up to 20 members with a 25% or higher representation
+The Editorial Board may expand to up to 20 members with a 25% or higher representation
 of community who are not core developers.
 
-The Workgroup members will act as the Editorial Board and will make decisions
+The Editorial Board will make decisions
 on controversial content, if needed.
 
-Although not part of this workgroup, a Documentation Team will also be created.
+Although not part of this Editorial Board, a Documentation Team will also be created.
```

Comment:
Again on making this current -- the documentation team I don't think exists (given current nomenclature), but we do have the documentation community and docs triagers.

A